index.js

```diff
@@ -1,159 +1,98 @@
-"use strict"
+'use strict';
 
 module.exports = {
-    "extends": "stylelint-config-standard",
-    "customSyntax": "scss",
-    "plugins": [
-        "stylelint-scss"
-    ],
-    "ignoreFiles": [
-        "sass/vendor/*.scss"
-    ],
-    "rules": {
-        "at-rule-empty-line-before": [
-            "always",
+    'extends': 'stylelint-config-standard-scss',
```

Comment:
There are in general a lot of overwrites. Do we really need them?

Reply:
The good think about is that if some new rule is added, it maybe already got a recommendation / standard when extending from this rulesets, else somebody always have to keep uptodate with changes of stylelint.